Expensify/App
# Fix: Added emoji category translations

### Details

- Added translations for emoji categories
- Added Food & Drinks category 
- Removed hardcoded spaces in the list and added a dynamic spacing logic.

### Fixed Issues

$ https://github.com/Expensify/App/issues/7060
$ https://github.com/Expensify/App/issues/7074

### Tests
1. Tested for English category names
2. Tested for Spanish category names



Errors are not related to my code

### QA Steps
1. Go to home page, and settings, change the language preference to Spanish
2. Go to any chat, and click on emoji button on compose input
3. Category names should have translated values 



### Tested On


### Screenshots


#### Web


**Translation** 
<img width="396" alt="web-emoji-cat-name-es" src="https://user-images.githubusercontent.com/3069065/148920146-2e37846a-464c-4456-940c-3c944c6a4b28.png">
<img width="434" alt="web-emoji-cat-name-eng" src="https://user-images.githubusercontent.com/3069065/148920155-7d595534-a44c-48a9-a340-0b21ee954fe5.png">

**Food & Drinks**
<img width="348" alt="web-food-drinks" src="https://user-images.githubusercontent.com/3069065/148937765-a5b648c9-19b8-4769-8c9d-9d3b47844958.png">

----------------------------------------------------------------------------------------

#### Mobile Web

**Translation**
<img width="360" alt="mweb-emoji-cat-name-es" src="https://user-images.githubusercontent.com/3069065/148920238-10b413f6-1c0e-40ca-b989-ceea0de13a9d.png">

**Food and Drinks**
<img width="336" alt="mweb-food-and-drinks" src="https://user-images.githubusercontent.com/3069065/148938733-a8fe98a5-2342-4890-b0a3-f3601b00e58d.png">

----------------------------------------------------------------------------------------

#### Desktop


**Translation**
<img width="453" alt="desktop-emoji-cat-name-es" src="https://user-images.githubusercontent.com/3069065/148920269-7341fec0-eef5-4f1e-b843-09d958470edc.png">

**Food and Drinks**
<img width="459" alt="desktop-food-and-drinks" src="https://user-images.githubusercontent.com/3069065/148939645-452ed201-a7f2-482b-83ef-eeff86d9d848.png">


----------------------------------------------------------------------------------------
#### iOS


**Translations**
<img width="343" alt="ios-emoji-cat-name-es" src="https://user-images.githubusercontent.com/3069065/148924212-d9a24565-cc61-4db2-818b-165cbe170e60.png">

**Food and Drinks**
<img width="377" alt="ios-food-and-drinks" src="https://user-images.githubusercontent.com/3069065/148939018-9dc7f9f4-b1f5-4e12-b229-a9d0de48f66d.png">

----------------------------------------------------------------------------------------
#### Android

**Translations**
<img width="330" alt="android-emoji-cat-name-es" src="https://user-images.githubusercontent.com/3069065/148921218-7d3c724a-1b84-48bf-b046-46628c5ac211.png">

**Food and Drinks**
<img width="329" alt="android-food-and-drinks" src="https://user-images.githubusercontent.com/3069065/148939073-f415e6dc-8956-4bc6-859f-5de810211dc2.png">
